src/main/java/com/example/gsoc/model/Plugin.java

```diff
@@ -0,0 +1,67 @@
+package com.example.gsoc.model;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "plugins")
+public class Plugin {
```

Comment:
Do we need `equals/hashCode` implementation to have a good integration with JPA?

Reply:
Sure, just added the autogenerated ones by my IDE.